client/js/screens/AboutUs/styles.js

```diff
@@ -0,0 +1,48 @@
+import {StyleSheet} from 'react-native';
+import {THEME} from '../../config';
+const {
+  colors: {black, darkGrey},
+  formatting: {baseSpacing, centeredChildren},
+  typography: {headerFont, hugeText, contentText, mainFont, subHeaderText},
+} = THEME;
+
+const styles = StyleSheet.create({
+  page: {...centeredChildren},
+  container: {
+    ...centeredChildren,
+    width: 308,
+  },
+  image: {
+    resizeMode: 'contain',
+    height: 131,
+    width: 174.67,
```

Comment:
use %s for height and width.

Reply:
AboutUs is moved to About, AboutUs page is deleted